main.go

```diff
@@ -21,8 +21,9 @@ var (
 	internalRecords []cloudflare.DNSRecord
 	upstreamRecords []cloudflare.DNSRecord
 	zoneId          string
+	zoneName        string
 	apiToken        string
-	quit                        = make(chan struct{})
+	quit            = make(chan struct{})
```

Comment:
It looks like this was unused, can you please remove it?

Reply:
I removed it, but that was something present before my addition